Model/ChargedProcessor.php

```diff
@@ -69,8 +76,19 @@ public function processChargedWebhook($params, $order)
             if (!$isBalancepayAuthCheckout) {
                 $orderPayment->capture(null);
             }
+            $orderPayment->setTransactionId($chargeId);
```

Comment:
shouldn't this be the Balance transaction id?

Reply:
We will remove this as its not required to be saved on payment object.

Reply:
Yes its removed, as we don't need transaction id saved, as Magento already saving it internally.